docs/additional-deployment-options/share-s3-access-logs.md

```diff
@@ -0,0 +1,19 @@
+---
+id: share-s3-access-logs
+title: Share S3 Access Logs
+hide_title: true
+---
+
+# Sharing S3 Access Logs
+
+In the NGAP environment, the ESDIS Metrics team has set up an ELK stack to process logs from Cumulus instances.  One step is to deliver any S3 Server Access logs that Cumulus creates.  We have provided a simple node package to aid this process.
```

Comment:
I'm not quite sure what you mean by `one step`. Are there more steps to the process?

Reply:
The first step?

Reply:
I meant it as "one step is to deliver s3 access logs" . Another is to set up cloudwatch logs sharing.  But I'll see if I can reword.

Reply:
Lauren took care of this in her PR.